packages/components/buttons/accessible-button/src/accessible-button.tsx

```diff
@@ -142,11 +142,15 @@ const AccessibleButton = forwardRef<HTMLButtonElement, TAccessibleButtonProps>(
       buttonProps = {
         type: props.type,
       };
-    } else if (!props.isDisabled) {
+    } else {
+      if (!props.isDisabled) {
+        buttonProps = {
+          onKeyPress: handleKeyPress,
+        };
+      }
       buttonProps = {
         role: 'button',
         tabIndex: '0',
-        onKeyPress: handleKeyPress,
       };
```

Comment:
Now it seems the element will never have the `onKeyPress` prop since we're overwriting the `buttonProps` variable after the `if` statement, wiping out that property.

Reply:
I let the `onkeyPress` callback check for the disabled property instead.
Updated here: https://github.com/commercetools/ui-kit/pull/2262/commits/96d4627e36fa2cd0363e666a8e6abfbe6be9473b